rlclientlib/logger/event_logger.cc

```diff
@@ -2,37 +2,82 @@
 #include "ranking_event.h"
 #include "err_constants.h"
 #include "time_helper.h"
+
+#include <functional>
 namespace reinforcement_learning { namespace logger {
+using namespace std::placeholders;
```

Comment:
the loggers that don't use the generic event's are v1 loggers (right @ataymano ?) Is this change mean to affect v1 and v2 logging, or just v2?